packages/viewport-react/src/types.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module '@automattic/viewport-react' {
+	export const useMobileBreakpoint: () => boolean;
+}
```

Comment:
Hey @scinos, this is the quickest solution that we found to move types, without introducing too much change in `viewport-react`.
The PR solves the code style issues and the viewport-based reference problem, so we're going to ship it. But it may be followed up by a better implementation of types in `viewport-react` 😄

Reply:
Unfortunately, we encountered some problems with tests with this solution, so we reverted to the old one.
We plan to deploy tomorrow in the afternoon (UTC), as this PR solves the code style and viewport problems, but we may follow up with another PR to improve the viewport-react types 😄